tests/test_config.py

```diff
@@ -0,0 +1,173 @@
+# pylint: disable=missing-docstring,redefined-outer-name
```

Comment:
Ah I should actually remove the `missing-docstring` one.
But the `redefined-outer-name` should actually be set for the whole `tests` subpackage, because it's how fixtures work in pytest: each test function/method's argument shadows an outer name.

Reply:
I removed the `missing-docstring` lint, but as said above I'd like to keep the `redefined-outer-name` one, if that's ok for you.

Reply:
ok, thanks for adding a comment in code explaining the globally disabled warning!